velox/exec/SortBuffer.cpp

```diff
@@ -285,11 +285,13 @@ void SortBuffer::spillOutput() {
 }
 
 void SortBuffer::prepareOutput(uint32_t maxOutputRows) {
```

Comment:
Did @FelixYBW common about needing to change the type of maxOutputRows to vector_size_t? If so, I agree.

Reply:
Looks the purpose of vector_size_t is to use as data type of batch size. But currently there are lots of violation in the code.